generated/nidaqmx/_task_modules/channels/ci_channel.py

```diff
@@ -1211,7 +1211,7 @@ def ci_count_edges_count_reset_logic_lvl_behavior(self):
         check_for_error(error_code)
 
     @property
-    def ci_count_edges_count_reset_reset_cnt(self):
+    def ci_count_edges_count_reset_reset_count(self):
```

Comment:
this is fine, but add a deprecation like we did with [ai](https://github.com/ni/nidaqmx-python/blob/master/generated/nidaqmx/_task_modules/channels/ai_channel.py#L9379)

Reply:
Added deprecation for the `ci_count_edges_count_reset_reset_cnt`